README.md

```diff
@@ -1,56 +1,162 @@
-# che-editor-intellij-community
+# JetBrains Projector Editor Images
 
-Che editor using intellij community edition
+Contains utilites and scripts, which allows to run JetBrains products in Che infrastructure using [Projector](https://github.com/JetBrains/projector-server).
 
+Projector is a server-side libraries, that allows to run Swing applications remotely.
 
-## build image
-```bash
-$ docker build -t che-editor-intellij .
+
+
+## Run JetBrains IDE in Eclipse Che
+
+In order to run JetBrains IDE in Eclipse Che infrastructure, current repository contains workspace configuration, which provides the ability to do that. To run the workspace you can use either Factory Link or create the Workspace from listed Devfiles below:
+
+##### IntelliJ IDEA Community
+
+- Che Factory pattern:
+
+  ```
+  https://<your-che-host>/f?url=https://github.com/che-incubator/jetbrains-editor-images/raw/master/devfiles/projector-idea-c.yaml
+  ```
+
+- Create the Workspace from [chectl](https://github.com/che-incubator/chectl/) (Requires clone the current repository):
+
+  ```sh
+  $ git clone https://github.com/che-incubator/jetbrains-editor-images && cd jetbrains-editor-images
+  $ chectl workspace:create -f devfiles/projector-idea-c.yaml
+  ```
+
+- Create the Workspace from the following workspace configuration:
+
+  ```yaml
+  metadata:
+    name: projector-idea-community
+  components:
+    - type: cheEditor
+      reference: 'https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-projector-idea-c.meta.yaml'
+      alias: projector-idea-c
+  apiVersion: 1.0.0
+  ```
+
+##### PyCharm Community
+
+- Che Factory pattern:
+
+  ```
+  https://<your-che-host>/f?url=https://github.com/che-incubator/jetbrains-editor-images/raw/master/devfiles/projector-pycharm-c.yaml
+  ```
+
+- Create the Workspace from [chectl](https://github.com/che-incubator/chectl/) (Requires clone the current repository):
+
+  ```sh
+  $ git clone https://github.com/che-incubator/jetbrains-editor-images && cd jetbrains-editor-images
+  $ chectl workspace:create -f devfiles/projector-pycharm-c.yaml
+  ```
+
+- Create the Workspace from the following workspace configuration:
+
+  ```yaml
+  metadata:
+    name: projector-pycharm-community
+  components:
+    - type: cheEditor
+      reference: 'https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-projector-pycharm-c.meta.yaml'
+      alias: projector-pycharm-c
+  apiVersion: 1.0.0
+  ```
+
+
+
+## Run JetBrains IDE in Docker
+
+In order to run JetBrains IDE in Docker, it is enough to pull one from two images, which publicitly deployed to [quay.io](https://quay.io/).
+
+At this moment it is community version for **IntelliJ IDEA** and **PyCharm**.
+
+```sh
+$ docker pull quay.io/che-incubator/che-projector-idea-c
```

Comment:
this image doesn't exist yet (or it's 403'd?)

Reply:
Correct, these images hasn't been pushed yet. They will be in place on push (github action) event after merging current PR.